### PR TITLE
Fix #7597, minor changes to enum_messages

### PR DESCRIPTION
**Pull request by wvu**
Hack and slash. Via IRC.

cc @geckom

### CODE REVIEW

File: modules/post/osx/gather/enum_messages.rb [hunk 1]
```diff
@@ -12,33 +12,65 @@ def initialize(info={})
...
         ],
       'DefaultAction' => 'ALL'
     ))
 
     register_options(
       [
         OptInt.new('MSGCOUNT', [false, 'Number of latest messages to retrieve.', 3]),
-        OptString.new('USER', [false, 'Username to retrieve messages from (defaults to current user)', 'CURRENT'])
-      ], self.class)
+        OptString.new('USER',  [false, 'Username to retrieve messages from (defaults to current user)'])
```
Comment (wvu-r7) [inline modules/post/osx/gather/enum_messages.rb]: Refactored to remove ```CURRENT```.

File: modules/post/osx/gather/enum_messages.rb [hunk 2]
```diff
@@ -112,80 +143,4 @@ def save(data)
...
-    end
-
-    results
-  end
-
-  #
-  # This is just a wrapper for cmd_exec(), except it chomp() the output,
-  # and retry under certain conditions.
-  #
-  def exec_shell_cmd(cmd)
```
Comment (wvu-r7) [inline modules/post/osx/gather/enum_messages.rb]: Switched to plain ```cmd_exec```.

File: modules/post/osx/gather/enum_messages.rb [hunk 3]
```diff
@@ -112,80 +143,4 @@ def save(data)
...
-      vprint_error("#{peer} - #{e.message} - retrying...")
-      retry
-    rescue EOFError => e
-      vprint_error("#{peer} - #{e.message} - retrying...")
-      retry
-    end
-  end
-
-  #
-  def locate_messages(base)
```
Comment (wvu-r7) [inline modules/post/osx/gather/enum_messages.rb]: Removed.

**Comment by bcook-r7** [comment]
## Release Notes

This improves the readability and coding style of the post/osx/gather/enum_messages module.

**Pull request merged**